pkg/apis/openebs.io/v1alpha1/cas_template_keys.go

```diff
@@ -103,6 +109,17 @@ const (
 	// {{- .ListItems.volumes.openebs.mypv.ip -}}
 	// {{- .ListItems.volumes.openebs.mypv.status -}}
 	ListItemsTLP TopLevelProperty = "ListItems"
+
+	// UpgradeItemTLP is a top level property supported by CAS template engine
+	//
+	// The properties provided by the caller are placed with UpgradeItemTLP
+	// as the top level property
+	//
+	// NOTE:
+	//  CAS template engine cannot modify these properties. These are the
+	// runtime properties that are provided as inputs to CAS template
+	// engine.
+	UpgradeItemTLP TopLevelProperty = "UpgradeItem"
```

Comment:
Can we avoid this.
upgrade engine code should take care of this.

Reply:
removed